src/JsonLd/Serializer/JsonLdEncoder.php

```diff
@@ -11,17 +11,32 @@
 
 namespace ApiPlatform\Core\JsonLd\Serializer;
 
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Serializer\Encoder\DecoderInterface;
+use Symfony\Component\Serializer\Encoder\EncoderInterface;
+use Symfony\Component\Serializer\Encoder\JsonDecode;
+use Symfony\Component\Serializer\Encoder\JsonEncode;
 use Symfony\Component\Serializer\Encoder\JsonEncoder;
 
 /**
  * JSON-LD Encoder.
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-final class JsonLdEncoder extends JsonEncoder
+final class JsonLdEncoder implements EncoderInterface, DecoderInterface
```

Comment:
Why can't we use `JsonEncoder` directly for JSON-LD? There is no special handling whatsoever required.